bitnami/prestashop/templates/deployment.yaml

```diff
@@ -85,36 +161,22 @@ spec:
             - name: PRESTASHOP_LANGUAGE
               value: {{ .Values.prestashopLanguage | quote }}
             {{- end }}
-            {{- if .Values.mariadb.enabled }}
-            - name: MARIADB_HOST
-              value: {{ template "prestashop.mariadb.fullname" . }}
-            - name: MARIADB_PORT_NUMBER
-              value: "3306"
+            - name: PRESTASHOP_DATABASE_HOST
+              value: {{ include "prestashop.databaseHost" . | quote }}
+            - name: PRESTASHOP_DATABASE_PORT_NUMBER
+              value: {{ include "prestashop.databasePort" . | quote }}
             - name: PRESTASHOP_DATABASE_NAME
-              value: {{ .Values.mariadb.db.name | quote }}
+              value: {{ include "prestashop.databaseName" . | quote }}
             - name: PRESTASHOP_DATABASE_USER
-              value: {{ .Values.mariadb.db.user | quote }}
+              value: {{ include "prestashop.databaseUser" . | quote }}
             - name: PRESTASHOP_DATABASE_PASSWORD
               valueFrom:
                 secretKeyRef:
-                  name: {{ template "prestashop.mariadb.fullname" . }}
+                  name: {{ include "prestashop.databaseSecretName" . }}
                   key: mariadb-password
-            {{- else }}
```

Comment:
externaldb-secrets.yaml doesn't work now

Reply:
Hi could you open an issue and explain it there?

Reply:
Hi, the external database is using a different password key
externaldb-secrets.yaml create: secrets
name: "common.names.fullname-externaldb" 
key: db-password

path:
            {{- if .Values.mariadb.enabled }}
            - name: PRESTASHOP_DATABASE_HOST
              value: {{ include "prestashop.databaseHost" . | quote }}
            - name: PRESTASHOP_DATABASE_PORT_NUMBER
              value: {{ include "prestashop.databasePort" . | quote }}
            - name: PRESTASHOP_DATABASE_NAME
              value: {{ include "prestashop.databaseName" . | quote }}
            - name: PRESTASHOP_DATABASE_USER
              value: {{ include "prestashop.databaseUser" . | quote }}
            - name: PRESTASHOP_DATABASE_PASSWORD
              valueFrom:
                secretKeyRef:
                  name: {{ include "prestashop.databaseSecretName" . }}
                  key: mariadb-password
            {{- else }}
            - name: PRESTASHOP_DATABASE_HOST
              value: {{ include "prestashop.databaseHost" . | quote }}
            - name: PRESTASHOP_DATABASE_PORT_NUMBER
              value: {{ include "prestashop.databasePort" . | quote }}
            - name: PRESTASHOP_DATABASE_NAME
              value: {{ include "prestashop.databaseName" . | quote }}
            - name: PRESTASHOP_DATABASE_USER
              value: {{ include "prestashop.databaseUser" . | quote }}
            - name: PRESTASHOP_DATABASE_PASSWORD
              valueFrom:
                secretKeyRef:
                  name: {{ printf "%s-%s" .Release.Name "externaldb" }}
                  key: db-password
            {{- end }}

Reply:
I've just create a PR https://github.com/bitnami/charts/pull/4026, please, let's move the conversation there. Thanks